include/rtc.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file rtc.h
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
```

Comment:
Do users need to call controller_init() to be able to use rtc functions?

Reply:
Nope. `controller_init` is only needed for actual controller inputs, not SI peripherals.

Reply:
Good